src/modules/micrortps_bridge/micrortps_client/microRTPS_client_main.cpp

```diff
@@ -100,6 +107,16 @@ static int parse_options(int argc, char *argv[])
 		}
 	}
 
+	if (_options.sleep_ms < 1) {
+		_options.sleep_ms = 1;
+		PX4_ERR("sleep time too low, using 1 ms");
+	}
+
+	if (_options.poll_ms < 1) {
+		_options.poll_ms = 1;
+		PX4_ERR("poll timeout too low, using 1 ms");
+	}
+
 	return 0;
```

Comment:
I first thought there is no other sleep in the same loop, so 0 would lead to a busy loop using full CPU. Now I see there is another usleep. What we should do is (in case of UART):
- remove the usleep & only use poll
- use a fixed timeout for poll, eg. 20ms. It does not need to be user-configurable.